packages/cst-dts-gen/src/api.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseParser } from "chevrotain"
+import { buildModel } from "./model"
+import { genDts, GenDtsOptions } from "./generate"
+
+const defaultOptions: GenDtsOptions = {
+  includeTypes: true,
+  includeVisitorInterface: true,
+  visitorInterfaceName: "ICstNodeVisitor"
+}
+
+export function generateCstDts(
+  parser: BaseParser,
+  options?: GenerateDtsOptions
+): string {
+  const effectiveOptions: GenDtsOptions = {
+    ...defaultOptions,
+    ...options
+  }
+
+  const productions = parser.getGAstProductions()
+  const model = buildModel(productions)
+
+  return genDts(model, effectiveOptions)
```

Comment:
should we add a newline at the end?

Reply:
Yeah, probably should.  The main use case would be to write the content to a file after all.